cpp/src/aztec/honk/composer/standard_honk_composer.test.cpp

```diff
@@ -281,19 +284,50 @@ TEST(standard_honk_composer, test_verification_key_creation)
     EXPECT_EQ(verification_key->permutation_selectors.size(), composer.program_width * 2);
 }
 
-TEST(StandarHonkComposer, BaseCase)
+TEST(StandardHonkComposer, BaseCase)
 {
     auto composer = StandardHonkComposer();
-    fr a = fr::one();
-    composer.circuit_constructor.add_public_variable(a);
+    fr a = 1;
+    composer.circuit_constructor.add_variable(a);
```

Comment:
This could be reset to public, but why bother until we handle public inputs in Adrian's upcoming PR?